test-utils/jest.setup.ts

```diff
@@ -0,0 +1,21 @@
+import { InternalDiscordManager } from './setup';
+import { server } from './mock-server';
+
+process.env.DISCORD_BOT_TOKEN = 'FAKE_BOT_TOKEN';
```

Comment:
I know you caught this one in the GitHub workflow, but this name should change too.
`process.env.DISCORD_TOKEN`
I pulled the PR to a clean repo and didn't have a .env file, so it failed the tests because of this.

Reply:
You're absolutely right. Good catch!